### ISSUE TITLE
where can i download the Beverly_hills image set from the csv?

### ISSUE THREAD

**Issue by nevton** [emadboctorx/yolov3-keras-tf2#4]

**Comment by nevton** [comment]
where can i download the Beverly_hills image set from the csv?

**Comment by emadboctorx** [comment]
It's a very small dataset(500 images) for testing purposes that I created here's a link to it https://drive.google.com/open?id=102bYRZYaOYwQMqpZYr5DgjpYikOmrE7x

**Comment by Eddddddddy** [comment]
That's an amazing repository. I want to try it on my computer. But there is some error with me. I guess it is because of a problem with my dataset.  Maybe https://drive.google.com/open?id=102bYRZYaOYwQMqpZYr5DgjpYikOmrE7x This can give me some help. But the labels link is damage.

**Comment by emadboctorx** [comment]
@Eddddddddy the link works perfectly fine

**Comment by emadboctorx** [comment]
@Eddddddddy https://drive.google.com/file/d/1IExWSL6_2F4ix8Xd0CKIUcF4I8pIz4wr/view?usp=sharing

**Comment by Eddddddddy** [comment]
I have sent a request to you. thank you so much. that will be very useful for me!

**Comment by semihcanturk** [comment]
Hi, is there an available test set anywhere now? I was attempting with VOC 2012 but had issues with setting up the dataset (due to missing voc cfg file), so was looking for a small dataset that's more or less ready for training. Thank you!
